### ISSUE TITLE
code readability 

### ISSUE THREAD

**Issue by Yuliya-Kaleda** [MadelynTav/MemeProject#30]
Many methods contain a large peace of code, which is hard to read and digest sometimes. For example, EditPhoto Activity has a super huge onCreate. Good practice to break such blocks in smaller methods.

**Comment by MadelynTav** [comment]
@Yuliya-Kaleda yes, we agree. We were so focused on features we didn't revise our code but we definitely should
